src/main/java/com/synopsys/integration/alert/component/settings/SettingsDescriptorActionApi.java

```diff
@@ -66,7 +66,38 @@ public void validateConfig(final Collection<ConfigField> descriptorFields, final
         super.validateConfig(descriptorFields, fieldModel, fieldErrors);
         validateLDAPSettings(fieldModel, fieldErrors);
     }
-    
+
+    @Override
+    public void testConfig(final Collection<ConfigField> descriptorFields, final TestConfigModel testConfig) throws IntegrationException {
+
```

Comment:
Should this be empty?

Reply:
yes there is no test button in the UI at the moment so this will remain empty.